src/openstudio_lib/LocationTabView.cpp

```diff
@@ -625,8 +632,122 @@ void LocationView::onWeatherFileBtnClicked() {
   }
 }
 
+std::vector<model::DesignDay> filterDesignDays(const std::vector<model::DesignDay>& designDays, const std::string& dayType, const std::string& percentage, const std::string& humidityConditionType = "") {
+  std::vector<model::DesignDay> filteredDesignDays;
+
+  std::copy_if(designDays.begin(), designDays.end(), std::back_inserter(filteredDesignDays), [&](const model::DesignDay& designDay) {
+  QString nameString = QString::fromStdString(designDay.name().get());
+
+    if (!nameString.contains("ann",Qt::CaseInsensitive))
+    { 
+      return false;
+    }
+
```

Comment:
@jmarrec somehow your other comments were lost but I addressed two of them in 

https://github.com/openstudiocoalition/OpenStudioApplication/pull/791/commits/78291c159fc5f6c17ae8ceae6b8a4a8edd7fd411

and https://github.com/openstudiocoalition/OpenStudioApplication/pull/791/commits/a5263ba7d166390d62bb03efac6e5f84a8a4262f